portfolio.html

```diff
@@ -4,10 +4,99 @@
     <meta charset="UTF-8">
     <title>portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <link rel="normalize" type="text/css>" href="normalize.css">
```

Comment:
type="text/css>" is incorrect.  Should be type="text/css"> note the position of the closing "

Reply:
Changed